docs/docs/connect/connect.md

```diff
@@ -0,0 +1,276 @@
+
+## Connection Strategies
+
+Rill offers flexible connection strategies to fit different data architectures and requirements.
+
+- ### _[Rill Managed OLAP + Data Ingestion (Default)](/connect/data-source)_:
+  
+  Use Rill's embedded **ClickHouse / DuckDB** (depending on size of data) as the OLAP engine and ingest data from external sources.  Full Rill functionality with some caveats depending which embedded engine you select. 
+ 
+      :::tip Rill Defaults with DuckDB
+      When starting Rill for the first time, Rill will autopopulate the connector with a `duckdb.yaml`. To use ClickHouse, create a managed ClickHouse connector, `clickhouse.yaml` with `managed: true`. For more information, see our [ClickHouse](/connect/olap/clickhouse) docs.
+
+      :::
+
+- ### _[Bring Your Own OLAP (BYO OLAP)](/connect/olap)_: 
+  
+  Large-scale datasets (100GB+) or existing OLAP infrastructure Connect to existing **ClickHouse**, **Druid**, **Pinot**, or **MotherDuck** instances. Use Rill's "live connectors" to ingest data directly into your OLAP engines.
+
+    :::note Modeling on BYO-OLAP
+    Some modeling features may be limited depending on the engine.
+    :::
+
+## Supported OLAP Engines
+
+### DuckDB
+### ClickHouse
+### MotherDuck
+### Druid
+### Pinot
+
+<div className="connector-icon-grid">
+  <ConnectorIcon
+    icon={<img src="/img/connect/icons/Logo-DuckDB.svg" alt="DuckDB" />}
+    content="DuckDB is the default engine for Rill Developer."
+    link="/connect/olap/duckdb"
+    linkLabel="Learn more"
+    referenceLink="duckdb"
+  />
+
+  <ConnectorIcon
+    icon={<img src="/img/connect/icons/Logo-Clickhouse.svg" alt="ClickHouse" />}
+    content="High-performance columnar database for real-time analytics and data warehousing."
+    link="/connect/olap/clickhouse"
+    linkLabel="Learn more"
+    referenceLink="clickhouse"
+  />
+
+  <ConnectorIcon
+    icon={<img src="/img/connect/icons/Logo-Motherduck.svg" alt="MotherDuck" />}
+    content="Cloud-native DuckDB service for scalable analytics and data processing."
+    link="/connect/olap/motherduck"
+    linkLabel="Learn more"
+    referenceLink="motherduck"
+  />
+
+  <ConnectorIcon
+    icon={<img src="/img/connect/icons/Logo-Druid.svg" alt="Druid" />}
+    content="Real-time analytics database designed for high-performance OLAP queries."
+    link="/connect/olap/druid"
+    linkLabel="Learn more"
+    referenceLink="druid"
+  />
+
+  <ConnectorIcon
+    icon={<img src="/img/connect/icons/Logo-Pinot.svg" alt="Pinot" />}
+    content="Distributed OLAP datastore for real-time analytics and business intelligence."
+    link="/connect/olap/pinot"
+    linkLabel="Learn more"
+    referenceLink="pinot"
+  />
+    <ConnectorIcon
+    icon={<img src="/img/connect/icons/Headset.svg" alt="Pinot" />}
+    content="Don't see an OLAP engine that you're looking for? Chat with us about your use case!"
+    link="/contact"
+    linkLabel="Learn more"
+  />
```

Comment:
Nit: I wouldn't format this in the same way as the actual connectors. Maybe just a one-liner sentence below would be better. 

Related, it didn't seem consistent to me that only the OLAP engines that this "Contact us" tile, and all the other connector types did not.